test/Datadog.Trace.Tests/Logging/Log4NetLogProviderTests.cs

```diff
@@ -147,5 +150,36 @@ public void DisabledLibLogSubscriberDoesNotAddTraceData()
             Assert.DoesNotContain(CorrelationIdentifier.TraceIdKey, logEvent.Properties.GetKeys());
             Assert.DoesNotContain(LoggingProviderTestHelpers.CustomPropertyName, logEvent.Properties.GetKeys());
         }
+
+        /// <summary>
+        /// Lighweight JSON-formatter for Log4Net inspired by https://github.com/Litee/log4net.Layout.Json
```

Comment:
Typo: `Lighweight`

Reply:
Thanks! Fixed in d5ee4f74d9f408be86bf373b8fe45b40bc72f37f